terraform/groups/lambda/main.tf

```diff
@@ -16,4 +16,49 @@ terraform {
 
 provider "aws" {
   region = var.aws_region
-}
\ No newline at end of file
+}
+
+module "secrets" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/parameter-store?ref=1.0.360"
+
+  name_prefix = local.service_name
+  kms_key_id  = data.aws_kms_key.kms_key.id
+  secrets     = nonsensitive(merge(local.service_secrets, local.stack_secrets))
```

Comment:
This appears to create a different parameter push to the ssm service map functionality before. Could it be re-examined?